test/sandbox-test.js

```diff
@@ -72,65 +82,41 @@ describe("sinon.sandbox", function () {
         it("uses restorable clock", function () {
             this.sandbox.useFakeTimers();
 
-            assert.isFunction(this.sandbox.clock.restore);
+            assert.clock(this.sandbox.clock);
         });
 
-        it("passes arguments to sinon.useFakeTimers", function () {
-            var stub = sinon.stub(sinon, "useFakeTimers").returns({ restore: function () {} });
+        // Not sure how we can test this without monkey-patching or testing the implementation of lolex.
```

Comment:
There have been a few fixes in `lolex` related to `Date` and timers recently, so maybe try upgrading to `1.5.1` and see if that helps? I could also help out if you are short on time.